runtimes/onnx-runtime/development/Dockerfile

```
@@ -27,8 +27,8 @@ ENV ONNX_BACKEND="onnxruntime.backend.backend"
 RUN locale-gen en_US.UTF-8 && update-locale LANG=en_US.UTF-8
 
 # Install dependencies
-RUN pip3 install onnx
-RUN pip3 install ort-nightly
+RUN pip3 install onnx-weekly
```

We should be testing the development version of ONNX Runtime against the stable version of ONNX. We don't want the ONNX tests to be changing before a release.
```suggestion
RUN pip3 install onnx
```

---

updated the docker file according to the comments and tested it.

For me it runs now again:

"======== 144 failed, 1173 passed, 1317 deselected, 1 warnings in 27.82s ========"